app/build.gradle

```diff
@@ -1,6 +1,7 @@
 plugins {
     id 'com.android.application'
     id 'org.jetbrains.kotlin.android'
+    id 'kotlin-kapt'
 }
 
 android {
```

Comment:
Please use a compile Sdk that has been released to have the documentation and source code for it.